test/built-ins/Array/fromAsync/asyncitems-bigint.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
I'd suggest maybe putting all the builtins-as-arraylikes in the same file?

Reply:
I'd rather not, since they're relying on messing with global state.